src/momento/__init__.py

```diff
@@ -1,5 +1,7 @@
 import logging
 
+from .auth import CredentialProvider
+from .config import configurations
```

Comment:
This is an oddball. Can it be Configurations?

Reply:
I hear you. We'd have to define an outer class in that module that contains all the pre-built configurations

Reply:
Refactored in [afcef14](https://github.com/momentohq/client-sdk-python/pull/295/commits/afcef14a8c9787d05727dffb3131c47679c8fa78)